research/src/sources/fast.json5

```diff
@@ -8,7 +8,29 @@
   "components": [
     {
       "name": "Accordion",
-      "url": "https://explore.fast.design/components/fast-accordion"
+      "url": "https://explore.fast.design/components/fast-accordion",
+      "concepts": [
+        {
+          "name": "collapsed",
+          "description": "4 closed items",
+          "image": "fast-accordion-collapsed.png"
+        },
+        {
+          "name": "open",
+          "description": "1 opened item and 3 closed",
+          "image": "fast-accordion-open.png"
+        },
+        {
+          "name": "collapse",
+          "description": "No restraining in the amount of collapsed items",
+          "image": "fast-accordion-collapse.gif"
+        },
+        {
+          "name": "accordion",
+          "description": "Only one opened item",
```

Comment:
Technically there both multi and single are supported under the `mode` attribute.